conf/modules.config

```
@@ -380,7 +380,7 @@ process {
             [
                 path: { "${params.outdir}/samtools/fastq" },
                 mode: params.publish_dir_mode,
-                pattern: '*.fastq.gz',
+                pattern: '*.unmapped_other.fastq.gz',
```

```suggestion
                pattern: '*._other.fastq.gz',
```

Maybe use the same pattern as in the [module ](https://github.com/nf-core/modules/blob/master/modules/nf-core/samtools/fastq/main.nf#L18)

---

Thanks. I will change it into `*_other.fastq.gz`